superset/config.py

```diff
@@ -500,6 +500,9 @@ class D3Format(TypedDict, total=False):
     # returned from each database in the ``SUPERSET_META_DB_LIMIT`` configuration value
     # in this file.
     "ENABLE_SUPERSET_META_DB": False,
+    # Set to True to replace Selenium library with Playwright to execute reports and thumbnails.
+    # Unlike Selenium, Playwright reports support deck.gl visualizations
```

Comment:
you may want to add here or in UPDATING.md or FEATURE_FLAGS.md that this flag will require the playwright pip package.